froglike6/data_structure/9012.py

```diff
@@ -0,0 +1,21 @@
+import sys
+input = sys.stdin.readline
+
+for _ in range(int(input())):
+    ps = input().rstrip()
```

Comment:
```suggestion
    T = input().rstrip()
```
입력을 받는 변수 이름을 ps로 하셨네요! 

저도 이전에 간단한 변수 이름을 사용했지만, 문제에서 주어지는 `T`라던가, `testCase` 정도로 어떤 변수인지 알 수 있게 작성하는 게 나중에 헷갈리지 않고 도움되는 것 같아요! 

(저는 문제에 있는 네임을 우선적으로 사용하는 편입니다😄)